### PR TITLE
MAT-6884 validate test case title and group does not have special cha…

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
…racters

## MADiE PR

Jira Ticket: [MAT-6884](https://jira.cms.gov/browse/MAT-6884)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/cms/gov/madie/measure/exceptions/SpecialCharacterException.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package cms.gov.madie.measure.exceptions;
+
+public class SpecialCharacterException extends RuntimeException {
+  private static final long serialVersionUID = 4113761000508937686L;
+
+  private static final String MESSAGE = "Test Case %s can not contain special characters";
```
Comment (adongare) [inline src/main/java/cms/gov/madie/measure/exceptions/SpecialCharacterException.java]: same here, I think we should let the caller know the restricted character set

Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/exceptions/SpecialCharacterException.java:6]: @adongare Changed too.

Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/exceptions/SpecialCharacterException.java]: @adongare Will do thanks Ashok!

Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/exceptions/SpecialCharacterException.java]: @adongare this is done, thanks!

**Review by adongare — APPROVED** [review]

**Pull request merged**